.github/workflows/deploy-prod.yml

```diff
@@ -48,18 +47,19 @@ jobs:
     name: Deploy to NAIS
     needs: build
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
     steps:
```

Comment:
Vet ikke om det har noe å si, men permissions kan flyttes på toppnivå, øverst i filen :)

Reply:
Ja, prøver her først, for å se om det "er nok"